pkg/util/computeskus/computeskus.go

```diff
@@ -77,3 +77,16 @@ func FilterVMSizes(skus []mgmtcompute.ResourceSku, location string) map[string]*
 
 	return vmskus
 }
+
+// SupportedOSDisk returns the type of OSDisk for the given resource. Most VMs will use Premium disks but some SKUs only support Standard SSDs
+func SupportedOSDisk(vmSku *mgmtcompute.ResourceSku) string {
+	const (
+		standardDisk          = "StandardSSD_LRS"
```

Comment:
We do not have any types for disks or capabilities so these are string constants.